src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -1967,15 +2567,29 @@ public enum RisingRivals implements LogicCardInfo {
             energyCost C
             attackRequirement {}
             onAttack {
-              damage 0
+              delayed {
```

Comment:
I would definitely make this BEFORE_LAST to ensure calculation happens in the right order.